backend/analytics_server/mhq/api/teams.py

```diff
@@ -151,7 +150,7 @@ def patch_team_repos_mapping(team_id: str, team_repos_data: List[TeamRepos]):
     for team_repo in team_repos_data:
         if team_repo.team_id != team_id:
             raise BadRequest(
-                f"Team Repo with repo_id: {team_repo.org_repo_id} team_id: {team_repo.team_id} does not match team in request url: {team_id}."
+                f"Team Repo with repo_id: {team_repo.org_repo_id} team_id: {team_repo.team_id} does not match team in request url: {team_id}."  # noqa E501
```

Comment:
Why `# noqa E501` here? These look like comments to suppress the linter.

Reply:
Just suppressing some linting rules. This one is for "line too long".
Seeing the codebase, I observe lots of places where lines are very long, refactoring all those makes no sense to me, as it would involve some code design change as well.

Not necessarily we  need need to follow all the standards of flake8.